instructor/common.py

```diff
@@ -1671,6 +1698,20 @@ def facs_histograms_edit(request, assignment, sample_prep, sp):
 
 
 @login_required
+def select_images(request):
+    mapping_id = request.POST.get('mapping_pk')
+    image_pk_list = request.POST.getlist('image_pk_list[]')
```

Comment:
This is how the attribute is passed in request.POST.
`request.POST.getlist('image_pk_list')` gives empty list.